prow/kube/cluster.go

```diff
@@ -70,8 +68,6 @@ func GetKubernetesClient(masterURL, kubeConfig string) (kubernetes.Interface, er
 	return client, nil
 }
 
-// GetKubernetesClient retrieves the Kubernetes cluster
-// client from within the cluster
 func GetProwJobClient(masterURL, kubeConfig string) (prowjobclientset.Interface, error) {
```

Comment:
missing a comment here?

Reply:
The old comment was wrong so I removed it until I add another one. It's updated now.